src/oemof/tabular/datapackage/reading.py

```diff
@@ -405,6 +455,37 @@ def find(n, d):
                 for f, v in facade.items():
                     if isinstance(v, Decimal):
                         facade[f] = float(v)
+                    # check if multi-period and value is list
+                    if period_data and isinstance(v, list):
+                        # check if length of list equals number of periods
+                        if len(v) == len(period_data["periods"]):
+                            if f in ["fixed_costs", "capacity_costs"]:
+                                # special period parameters don't need to be
+                                # converted into timeseries
+                                facade[f] = v
+                                msg = (
+                                    f"\nThe parameter '{f}' of a "
+                                    f"'{facade['type']}' facade is converted "
+                                    "into a periodic list. This might not be "
+                                    "possible for every parameter and lead to "
+                                    "ambiguous error messages.\nPlease be "
+                                    "aware, when using this feature!"
+                                )
+                                warnings.warn(msg, UserWarning)
```

Comment:
The message does not belong here, as paramter has not been converted.
Instead, I would replace whole part with simple `continue` as value does not has to be adapted...